cura/Machines/Models/DiscoveredPrintersModel.py

```diff
@@ -66,12 +66,23 @@ def readableMachineType(self) -> str:
 
     @pyqtProperty(bool, notify = machineTypeChanged)
     def isUnknownMachineType(self) -> bool:
-        return self.readableMachineType.lower() == "unknown"
+        return self.readableMachineType == catalog.i18nc("@label", "Unknown")
```

Comment:
We really shouldn't do this. Comparing with translations is usually a big no-no, because it's rather prone to breaking.

Reply:
Also; We could just ignore this situation, as it only happens in the UM office with UM3 prototypes.

Reply:
I removed the translation of "Unknown" here in 26205fbdc. I still think we should keep the "isUnknown" flag though, at least it will be more clear when we are testing in the UM office. Your call.

Reply:
I'm fine with keeping it in. It should be more defensive if a printer firmware has a bug, or if we're dealing with different discovery plug-ins such as OctoPrint or USB.